src/Atmos/Model/multiphysics_types.jl

```diff
@@ -119,3 +121,216 @@ function compute_rain_params(atmos, state, aux, t, ts)
 
     return (S_qt = S_qt, S_ql = S_ql, S_qr = S_qr, I_l = I_l, Φ = Φ)
 end
+
+export RainSnow_1M
+"""
+    RainSnow_1M{FT} <: AbstractSource
+A collection of source/sink terms related to 1-moment rain and snow microphysics
+"""
+struct RainSnow_1M{
+    PV <:
+    Union{Mass, Energy, TotalMoisture, LiquidMoisture, IceMoisture, Rain, Snow},
+} <: TendencyDef{Source, PV} end
+
+RainSnow_1M() = (
+    RainSnow_1M{Mass}(),
+    RainSnow_1M{Energy}(),
+    RainSnow_1M{TotalMoisture}(),
+    RainSnow_1M{LiquidMoisture}(),
+    RainSnow_1M{IceMoisture}(),
+    RainSnow_1M{Rain}(),
+    RainSnow_1M{Snow}(),
+)
```

Comment:
❤️

Reply:
It looks good. I think this source term change improves readability. Go refactor!